server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/main/java/org/apache/james/webadmin/integration/vault/LinshareBlobExportMechanismIntegrationTest.java

```diff
@@ -136,10 +136,9 @@ void exportShouldShareTheDocumentViaLinshareWhenJmapDelete() {
 
         exportVaultContent(webAdminApi, EXPORT_ALL_HOMER_MESSAGES_TO_USER_1);
 
-        assertThat(user1LinshareAPI.receivedShares())
+        assertThat(user1LinshareAPI.listAllDocuments())
             .hasSize(1)
-            .allSatisfy(receivedShare -> assertThat(receivedShare.getDocument().getName()).endsWith(".zip"))
-            .allSatisfy(receivedShare -> assertThat(receivedShare.getSender().getMail()).isEqualTo(USER_1.getUsername()));
```

Comment:
you didn't explained why you removed this assertion. What are the rationales?

Reply:
I believed this https://github.com/linagora/james-project/pull/3090#discussion_r376204331 will answer your question.

Reply:
It doesn't sorry.

> We just upload directly to UserSpace by delegation account then OpenPaas will do the send email.

If it sends an email, why don't we check that it's received?

Reply:
Yes, we do check `target user` received a Document at this `.hasSize(1)`.Also i can add 1 more condition about `filename`.

Reply:
You do not check it receives the email.

Reply:
Yes, we upload file to the `target user` then we check that `user` had `that file` in `myspace`, that's all right?

Reply:
No. You said:

> We just upload directly to UserSpace by delegation account then OpenPaas will do the send email.

So I assume the whole process actually sends a mail.

Why wouldn't we check that mail?

Reply:
Yes, i mentioned that OpenPaas will send the mail but we dont know when, how it did that.

And in this case, we have integration test with James and Linshare not OpenPaas so we just care about the document is received in target user `myspace` that all.

Reply:
Could you describe me the workflow of this feature: how linshare triggers OpenPaaS for this mail sending, which component of OpenPaaS, etc?

Reply:
Here I can help.

 - 1. OpenPaaS triggers the James task for export (superTrash or mailbox/backup)
 - 2. James retrieve data, zip them, save that to the blobStore
 - 3. James pushes the data to LinShare using a technical User and delegation
 - 4. OpenPaaS monitors the overall information via James execution task details (polling - unless we choose a better rabbitmq push based mechanism for instance)
 - 5. OpenPaaS then display progress, and sends a i8n mail/notification upon completion.

Sorry, this should be in the ticket, at least Linagora side.

Reply:
Ok, makes sense, thank you.

What's sad is that we go from a tested feature to an untested one.

Reply:
> What's sad is that we go from a tested feature to an untested one.

I do agree